packages/generate-pdf/test/generate-pdf.spec.ts

```diff
@@ -188,29 +181,26 @@ const runPDFGenerationTests = (testSuite: PDFGenerationTestProperties) => {
       const outputFilePath = `./${type}-${version}-test2.pdf`;
       const translations = JSON.parse(readFileSync(translationsPath, 'utf8'));
       const extraTranslations = extraTranslationsPath ? JSON.parse(readFileSync(extraTranslationsPath, 'utf8')) : {};
-      const generatePdfOptions: GeneratePdfOptionsExtended<'stream'> = isUnreleasedVersion
+      const generatePdfOptions: GeneratePdfOptions = isUnreleasedVersion
         ? {
             docDefinition: { ...docDefinition, styles },
-            outputType: 'stream',
             fonts,
             generatorPath,
             translations,
             extraTranslations,
             languageFontMap,
+            a3Compliant: true,
           }
         : {
             docDefinition,
-            outputType: 'stream',
             fonts,
             generatorPath,
             languageFontMap,
+            a3Compliant: true,
           };
-      //
+
       const pdfDoc = await generatePdf(validCertificate, generatePdfOptions);
-      const writeStream = createWriteStream(outputFilePath);
-      pdfDoc.pipe(writeStream);
-      pdfDoc.end();
-      await waitWritableStreamEnd(writeStream, outputFilePath);
```

Comment:
`waitWritableStreamEnd` contained an `expect` clause that checked the existence of the newly created file. Removing it means that check has also been removed.